NOTICES

```diff
@@ -5,15 +5,15 @@ This file details additional third party software license agreements
 and third party notices and information that are required 
 to be reproduced for the following programs:
    
-Operator for IBM Block Storage CSI Driver version 1.2.0
+Operator for IBM Block Storage CSI Driver version 1.3.0

```

Comment:
@ArbelNathan wasn't this supposed to be done as part of https://github.com/IBM/ibm-block-csi-operator/pull/95?